modules/data-loading/examples/config-avro

```diff
@@ -0,0 +1,57 @@
+connector.class=org.apache.kafka.connect.mirror.MirrorSourceConnector
+source.cluster.alias=Hello
+target.cluster.alias=World
+source.cluster.bootstrap.servers=source.kafka.server:9092
+target.cluster.bootstrap.servers=localhost:30002
+source->target.enabled=true
+topics=avro-without-registry-topic
+replication.factor=1
+sync.topic.acls.enabled=false
+checkpoints.topic.replication.factor=1
+heartbeats.topic.replication.factor=1
+offset-syncs.topic.replication.factor=1
+offset.storage.replication.factor=1
+status.storage.replication.factor=1
+config.storage.replication.factor=1
+emit.heartbeats.interval.seconds=5
+secondary.scheduled.rebalance.max.delay.ms=35000
```

Comment:
@ryantigergraph pointed out the field should be prefixed with target cluster alias `world`. It should be
```
world.scheduled.rebalance.max.delay.ms=35000
```